src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -484,13 +474,13 @@
 
         "isSuppressedInSource": 
         {
-          "description": "A flag that indicates whether or not this result was suppressed in source code.",
+          "description": "A flag indicating whether or not this result was suppressed in source code.",
           "type": "boolean"
         },
 
         "fixes": 
         {
-          "description": "An array of fix objects, if available, that can be applied in order to correct this result.",
+          "description": "An array of 'fix' objects, each of which represents a proposed fix to the problem indicated by the result.",
```

Comment:
ok, this suggests you are comfortable with direct reference to JSON names. I'm ok with this too if you want to go with it.